src/mn_menu.c

```diff
@@ -186,7 +186,7 @@ short whichSkull;              // which skull to draw (he blinks)
 
 char skullName[2][/*8*/ 9] = {"M_SKULL1", "M_SKULL2"};
 
-static menu_t SaveDef, LoadDef, LoadAutoSaveDef;
+static menu_t SaveDef, LoadDef, QuickLoadDef, LoadAutoSaveDef;
```

Comment:
Could you add a comment what's the difference between these four menus now?

Reply:
Added here: https://github.com/fabiangreffrath/woof/pull/1947/commits/0b9e61c44d4ade6b1344ba375e10c87bf52be691